module/VuFind/tests/unit-tests/src/VuFindTest/OAuth2/Repository/IdentityRepositoryTest.php

```diff
@@ -34,8 +34,8 @@
 use PHPUnit\Framework\MockObject\Exception;
 use PHPUnit\Framework\MockObject\MockObject;
 use VuFind\Auth\ILSAuthenticator;
+use VuFind\Db\Entity\User;
```

Comment:
Might be slightly more generic to use `UserEntityInterface` here instead of the concrete `User` class.